### PR TITLE
Quicklinks

### PR DESCRIPTION
**Pull request by ianmah**
- In an effort to keep PRs small I will do the rest of the quicklinks page in a different PR
- Changed Button to an `a` element, its better practice to make links that go to other pages be `a` elements
- Removed progress bar ;(
- Created CommonLinks component
- Live Button in sidebar

image

### CODE REVIEW

File: src/components/Common.js [hunk 1]
```diff
@@ -12,23 +12,25 @@ export const Card = styled.div`
...
 `
 
-export const Button = styled.button`
+export const Button = styled.a`
   ${text};
-  cursor: pointer;
+  display: block;
+  text-decoration: none;
   font-family: 'HK Grotesk';
+  font-weight: 600;
```
Comment (jackyzha0) [inline src/components/Common.js:20]: Let's try to use theme here? `${p => p.theme.typography.h2.weight}` is 600

File: src/components/Sidebar.js [hunk 2]
```diff
@@ -33,16 +33,42 @@ const StyledA = styled(A)`
...
+  padding: 5px;
+`
+
 export default () => {
   const [location] = useLocation();
 
   return (
     <SidebarContainer>
-      <Header>Some Title</Header>
+      <Header>nwHacks</Header>
```
Comment (jackyzha0) [inline src/components/Sidebar.js:61]: Thoughts on having this as a constant in `Constants.js`

File: src/containers/Countdown.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React, { useState, useEffect } from 'react';
 import styled from 'styled-components'
 import { H2 } from '../components/Typography'
 import TimeDisplay from '../components/TimeDisplay'
-import ProgressBar from '../components/ProgressBar'
+// import ProgressBar from '../components/ProgressBar'
```
Comment (jackyzha0) [inline src/containers/Countdown.js:5]: big sad

File: src/components/Common.js (hunk 1 above)
Comment (ianmah) [reply src/components/Common.js:20]: m e m e d yessir

File: src/components/Sidebar.js (hunk 2 above)
Comment (ianmah) [reply src/components/Sidebar.js:61]: Should we eventually put this in the cms? If yes I will leave this for now?

Comment (jackyzha0) [reply src/components/Sidebar.js:61]: Oh yeah, we should meet to talk about Livesite x CMS integration LOL I guess we can leave it as is rn

**Review by jackyzha0 — APPROVED** [review]
l g t m 🍄

**Pull request merged**